src/test/java/org/opentripplanner/middleware/persistence/PersistenceUtil.java

```diff
@@ -45,6 +54,36 @@ public static ApiUser createApiUser(String email) {
         return user;
     }
 
+    /**
+     * Create Admin user and store in database.
+     */
+    public static AdminUser createAdminUser(String email) {
+        AdminUser user = new AdminUser();
+        user.email = email;
+        Persistence.adminUsers.create(user);
+        return user;
+    }
+
+    /**
+     * Get Api user from database
+     */
+    public static ApiUser getApiUser(String id) {
+        return Persistence.apiUsers.getById(id);
```

Comment:
I'm not sure these one-line method calls really add much at this stage. If we were doing more in the method body it might make sense, but I feel like this just obfuscates things at the moment.

Reply:
Updated.